.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -14,7 +14,7 @@ on:
       test-cluster-name:
         required: true
         type: string
-      appsignals-adot-image-name:
+      application-signals-adot-image:
```

Comment:
Doesn't this impact workflows in other repos? Did you make PRs to aws-otel-java-instrumentation accordingly and check if there are any other places there may be impact?

Reply:
For `aws-otel-java-instrumentation`, this workflow isn't used, the java repo uses its [own copy of this workflow](https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/.github/workflows/appsignals-e2e-eks-test.yml#L14). In this repository on the other hand, this isn't used anymore. To be extra safe, I'll undo this change, keep it as is until we remove it entirely.